manifests/5.0/cluster-logging.v5.0.0.clusterserviceversion.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
     categories: "OpenShift Optional, Logging & Tracing"
     certified: "false"
     description: |-
-      The Cluster Logging Operator for OCP provides a means for configuring and managing your aggregated logging stack.
+      The RedHat OpenShift Logging Operator for OCP provides a means for configuring and managing your aggregated logging stack.
```

Comment:
@vimalk78 i think this should be "Red Hat"  I'll update the issue.

Reply:
updated